src/arrays.js

```diff
@@ -1,61 +1,72 @@
 const getNthElement = (index, array) => {
-  // your code here
+if (index > array.length -1) {
+  return array[index % array.length]
+} else { return array[index]}
 };
 
 const arrayToCSVString = (array) => {
-  // your code here
+  return array.toString()
 };
 
 const csvStringToArray = (string) => {
-  // your code here
+  return string.split(',')
 };
 
 const addToArray = (element, array) => {
-  // your code here
+   array.push(element)
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  return newArray2 = array.concat(element);
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+  array.splice(index, 1);
 };
 
 const numbersToStrings = (numbers) => {
-  // your code here
+  return numbers.map(strings => strings.toString());
 };
 
 const uppercaseWordsInArray = (strings) => {
-  // your code here
+  return strings.map(uppercase => uppercase.toUpperCase());
 };
 
 const reverseWordsInArray = (strings) => {
-  // your code here
+    return strings.map(reversed => reversed.split("").reverse().join(""));
 };
 
 const onlyEven = (numbers) => {
-  // your code here
+    return numbers.filter(number => number % 2 === 0 )
 };
 
 const removeNthElement2 = (index, array) => {
-  // your code here
+  return array.filter(arrFil => arrFil !== array[index]);
 };
 
 const elementsStartingWithAVowel = (strings) => {
-  // your code here
-};
+    return strings.filter(arrFil => arrFil.charAt(0) === 'a' ||
+    arrFil.charAt(0) === 'e' || arrFil.charAt(0) === 'i' || 
+    arrFil.charAt(0) === 'o' || arrFil.charAt(0) === 'u' || 
+    arrFil.charAt(0) === 'A' || arrFil.charAt(0) === 'E' || 
+    arrFil.charAt(0) === 'I' || arrFil.charAt(0) === 'O' ||
+    arrFil.charAt(0) === 'U')
+  };
 
 const removeSpaces = (string) => {
-  // your code here
+    return string.replace(/\s/g,'');
+  
 };
 
 const sumNumbers = (numbers) => {
-  // your code here
+      return numbers.reduce(function(total,curVal) {
```

Comment:
Definitely some wacky indentation within this block - always ensure 2 spaces inside every block of code you enter. If it were nested, it'd look something like this:

```js
const nestedCode = isNested => {
  if (isNested) {
    console.log('isNested is true')
    
    if (1 === 1) {
      console.log('Also, 1 is strictly equal to 1')
    }
  }

  return 'isNested is false'
}
```